quit/wsgi.py

```diff
@@ -13,5 +15,8 @@
 sys.setrecursionlimit(2 ** 15)
 application = create_app(config)
 
+# Set the basepath
+application.wsgi_app = DispatcherMiddleware('/', {parsedArgs.basepath: application.wsgi_app})
```

Comment:
Do not set DispatcherMiddleware if parsedArgs.basepath is not set.
This means, parsedArgs.basepath is None which in turn means the default value for parsedArgs.basepath should be set to None